diode-server/reconciler/logs_retriever.go

```diff
@@ -13,6 +13,7 @@ import (
 	"google.golang.org/protobuf/encoding/protojson"
 
 	"github.com/netboxlabs/diode/diode-server/gen/diode/v1/reconcilerpb"
+	"github.com/redis/go-redis/v9"
```

Comment:
nit: goimports

Reply:
i've run `goimports -l -w `, no changes happened..

Reply:
Let's treat `github.com/netboxlabs/diode` as local, i.e. with `goimprots -l -local "github.com/netboxlabs/diode" -w`.

The expected imports groups:
```
stdlib

3rd party libs

local libs
```

Reply:
cool. it fixed some files!